app/src/main/java/org/mozilla/rocket/download/DownloadInfoRepository.kt

```diff
@@ -80,8 +80,12 @@ class DownloadInfoRepository {
 
     fun deleteFromDownloadManager(downloadId: Long) {
 
+        DownloadInfoManager.getInstance().downloadManager.remove(downloadId)
+    }
+
+    fun trackDownloadCancel(downloadId: Long): Boolean {
```

Comment:
The returned boolean value is not used. Do we still need to have return value?

Reply:
fixed